pomodoro-client.py

```diff
@@ -34,7 +34,7 @@ def format_is_paused(is_paused):
 
 def format_state(state):
     return {
-        "pomodoro": "Pomodoro",
+        "pomodoro": "",
```

Comment:
what is ? How does this implement this:

> toggle now starts a pomodoro if none is active i.e. pomodoro.State = null

Reply:
I accidentally overwrote some code. If you close this I came up with something better in the meantime!